docs/workflow/trimming/feature-switches.md

```diff
@@ -8,35 +8,42 @@ configurations but their defaults might vary as any SDK can set the defaults dif
 
 | MSBuild Property Name | AppContext Setting | Description |
 |-|-|-|
+| AutoreleasePoolSupport | System.Threading.Thread.EnableAutoreleasePool | When set to true, creates an NSAutoreleasePool for each thread and thread pool work item on applicable platforms. |
```

Comment:
This is duplicating official documentation at https://learn.microsoft.com/en-us/dotnet/core/deploying/trimming/trimming-options?pivots=dotnet-8-0#trimming-framework-library-features . Should we just link to the official docs from here for the publicly documented feature switches instead of trying to keep the two docs in sync?